src/Npgsql/PublicAPI.Unshipped.txt

```diff
@@ -1,5 +1,6 @@
 ﻿#nullable enable
 abstract NpgsqlTypes.NpgsqlTsQuery.Equals(NpgsqlTypes.NpgsqlTsQuery? other) -> bool
+Npgsql.INpgsqlDatabaseInfoFactory.Load(Npgsql.Internal.NpgsqlConnector! conn, Npgsql.Util.NpgsqlTimeout timeout, bool async) -> System.Threading.Tasks.Task<Npgsql.NpgsqlDatabaseInfo?>!
```

Comment:
I think we need to move INpgsqlDatabaseInfoFactory into Internal as well, since it now depends on an internal type (NpgsqlConnector).